### ISSUE TITLE
Prompt to copy settings/data when upgrading to Osmand+

### ISSUE THREAD

**Issue by ejain** [osmandapp/Osmand#3374]
When running Osmand+ for the first time, and existing Osmand data is present, the app could prompt to import the existing settings and map data.

**Comment by vshcherb** [comment]
Agree, though importing settings is not straight forward cause those folders are private usage to the app itself.

**Comment by what-aboot** [comment]
If you cannot upgrade/transfer from one version to another, then OsmAnd should have stopped all it's config files and all of its tracks in a public directory. As it is, OsmAnd+ should be installed in the same  directory OR add the plus features as a .DLL onto the free version.

Android 12 completely denies access to Android directory, so you're saying I must lose all the tracks I have on free?
